src/electron/views/device-connect-view/components/android-setup/detect-adb-step.tsx

```diff
@@ -2,11 +2,25 @@
 // Licensed under the MIT License.
 import { NamedFC } from 'common/react/named-fc';
 import { CommonAndroidSetupStepProps } from 'electron/views/device-connect-view/components/android-setup/android-setup-types';
+import { Spinner, SpinnerSize } from 'office-ui-fabric-react';
 import * as React from 'react';
 
+import { androidSetupSpinner } from '../../device-connect-view.scss';
+
 export const DetectAdbStep = NamedFC<CommonAndroidSetupStepProps>(
     'DetectAdbStep',
     (props: CommonAndroidSetupStepProps) => {
-        return <>step: {props.androidSetupStoreData.currentStepId}</>;
+        const renderSpinner = () => {
+            return (
+                <Spinner
```

Comment:
It looks like this code creates a function on line 13 (`renderSpinner`) and then returns its result after calling it. We may be able to return the spinner directly, e.g. lines 14 - 22

Reply:
True; I'm not sure why I didn't just simply return the spinner. Thanks for noticing this